umh.docs.umh.app/content/en/docs/getstarted/managingthesystem.md

```diff
@@ -1,142 +1,247 @@
-  together, to provide a complete solution for your manufacturing needs. From 
-  data collection to contextualization and visualization, or data storage. 
-  For rare cases that require a special feature, you can add it to your UMH
-  stack by creating your own custom microservice.
+## A Few Words About the Communication
 
-## 1. Install the UMH
+Now that you have connected a UMH instance to the Management Console, you might
+be curious about how the Management Console communicates with the instance.
 
+The Management Companion is a microservice that runs in conjunction with the UMH.
+It plays a crucial role in linking the instance to the Management Console. This
+includes gathering the status of the device and the various components within
+the system.
 
-1. Download and install [Docker Desktop](https://www.docker.com/products/docker-desktop)
-   for your operating system. This is only 
-   necessary for local installations.
-   
+The diagram below illustrates the communication flow between the Management
+Console and the instance:
 
-2. Follow the tutorial in the Management Console to create your first local 
-   instance of the United Manufacturing Hub. Once the installation is finished,
-   press the blue **Finish** button the right bottom corner of the Console. 
+![Communication between the Management Console and the instance](/images/getstarted/managingTheSystem/simpleInfrastructureDiagram.png?width=80%)
 
+## Overview of Your Instances
 
-{{% notice info %}}
-A local installation functions similar to an installation on an external device,
-like an edge device or a VM. The benefit is, that you can easily set it up
-without any additional hardware and learn how to use it with simulated data.
+On the left side of the Management Console, you can view the list of your
+instances. If you have just installed the UMH, you should see only one instance
+in the list.
+
+The status of your instance is indicated by color: green means everything is
+working properly, while yellow indicates that there may be an issue.
+
+The **Messages Received** statistic shows the number of messages received by you
+from the instance since you opened the Management Console. It is usually a good
+indicator of the health of the Companion and the latency of the connection between
+you and the instance.
+
+## Monitoring the Instance's Status
+
+From the instance dashboard, in the overview tab, you can view the status of
+your instance. There are multiple interfaces that display the status of each
+component of the system.
+
+### Modules
+
+A _Module_ refers to a group of workloads in the United Manufacturing Hub
+responsible for specific tasks. For example, the _Data Source_ module collects
+data from various sources and is composed of multiple microservices.
+
+In the **Modules** tab, you can view the status of each module. If a module is
+not healthy, it means that one or more of its microservices are not functioning
```

Comment:
it is not only monitoring the microservices, it is acutally monitoring way more:
- is the service still reachable?
- are any errors or warnings in the logs?
etc.

As erxample I woul use the Historian. THe Historian consists out of TimescaleDB, which itself consists out of a storage, a docker container linked to the storage, and one or multiple services making it accessible outside of the cluster. In HA setups, there will actrually one primary instance, and then multiple read replica.